### ISSUE TITLE
DeepAD: Alzheimer′s Disease Classification via Deep Convolutional Neural Networks using MRI and fMRI

### ISSUE THREAD

**Issue by agitter** [greenelab/deep-review#86]
http://doi.org/10.1101/070441 (still processing, http://biorxiv.org/content/early/2016/08/21/070441)

**At a glance:** They use adopted versions of the LeNet and GoogleNet architectures to classify brain scans.  They appear to be pre-processing the imaging data into 2D instead of operating on the original dimension.  There are 144 resting state fMRI subjects and 302 MIR subjects (those counts are case + control), so there isn't much training data in terms of subjects, but there are many instances once they convert the scans to 2D.  They report high accuracy, but I don't know what to expect and they don't compare to any other methods on this dataset.  The accuracies they report for related previous work may not be relevant.  This paper does refer to many previous deep learning frameworks if we are interested in exploring this topic.  The topic itself, supervised learning for neuroimaging, is relevant to the review.

> To extract patterns from neuroimaging data, various techniques, including statistical methods and machine learning algorithms, have been explored to ultimately aid in Alzheimer's disease diagnosis of older adults in both clinical and research applications. However, identifying the distinctions between Alzheimer′s brain data and healthy brain data in older adults (age > 75) is challenging due to highly similar brain patterns and image intensities. Recently, cutting-edge deep learning technologies have been rapidly expanding into numerous fields, including medical image analysis. This work outlines state-of-the-art deep learning-based pipelines employed to distinguish Alzheimer′s magnetic resonance imaging (MRI) and functional MRI data from normal healthy control data for the same age group. Using these pipelines, which were executed on a GPU-based high performance computing platform, the data were strictly and carefully preprocessed. Next, scale and shift invariant low- to high-level features were obtained from a high volume of training images using convolutional neural network (CNN) architecture. In this study, functional MRI data were used for the first time in deep learning applications for the purposes of medical image analysis and Alzheimer′s disease prediction. These proposed and implemented pipelines, which demonstrate a significant improvement in classification output when compared to other studies, resulted in high and reproducible accuracy rates of 99.9% and 98.84% for the fMRI and MRI pipelines, respectively.

**Comment by gwaygenomics** [comment]
This is very similar to this [paper](http://dx.doi.org/10.1101/066910) which has even _less_ cases and controls (n = 28 and n = 15, respectively).

In both papers, they try to predict AD vs. healthy, and both have extremely high accuracy- a much more interesting (and difficult) question would be AD vs. dementia.
